src/components/MappingPanel.js

```
@@ -0,0 +1,82 @@
+    this.ul.style.listStyle = 'none';
+    this.ul.style.padding = '0';
+
+    this.container.appendChild(this.ul);
+    this.parent.appendChild(this.container);
+
+    this._onStoreChange = this._onStoreChange.bind(this);
+    this.unsubscribe = store.subscribe(this._onStoreChange);
+  }
+
+  _onStoreChange(newState, prevState) {
+    if (newState.schema !== (prevState && prevState.schema) || newState.mapping !== (prevState && prevState.mapping)) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)